### PR TITLE
Employee list + additional classes

### PR DESCRIPTION
**Pull request by OrestPokotylenko**

### CODE REVIEW

File: DAL/BaseDao.cs [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
-            
-            Test(client);
-        }
-
-        static async Task Test(MongoClient client)
-        {
-            // TEST
-            var database = client.GetDatabase("NoSQLCluster");
-            
+            var connectionString = ConfigurationManager.AppSettings["MongoDB"];
```
Comment (AnastasiaIU) [inline DAL/BaseDao.cs]: Please, follow the official documentation of MongoDB and declare the connection string outside of the BaseDao constructor as a const string.
https://www.mongodb.com/docs/drivers/csharp/current/fundamentals/connection/connect/

```
const string ConnectionString = ConfigurationManager.AppSettings["MongoDB"];
```

**Review by tina262317 — APPROVED** [review]

File: DAL/BaseDao.cs [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
+            var client = new MongoClient(connectionString);
```
Comment (AnastasiaIU) [inline DAL/BaseDao.cs]: Please, make the client static, as we disscused.

File: DAL/BaseDao.cs (hunk 1 above)
Comment (AnastasiaIU) [reply DAL/BaseDao.cs]: const doesn't work with the ConfigurationManager, so we will make it readonly.

File: DAL/BaseDao.cs [hunk 3]
```diff
@@ -1,28 +1,21 @@
...
+using Model;
 using MongoDB.Driver;
 
 namespace DAL
 {
     public class BaseDao
     {
-        string connectionString;
-        MongoClient client;
+        protected readonly IMongoCollection<Employee> _employeeCollection;
```
Comment (AnastasiaIU) [inline DAL/BaseDao.cs]: Please add this comment, so we will remember it's only references and not cached data, which is why it's readonly.

`// References to the collections in the MongoDB instance.`

**Review by AnastasiaIU — APPROVED** [review]

**Pull request merged**